cmd/terramate/cli/cli.go

```diff
@@ -321,26 +424,47 @@ func (c *cli) initStack(dirs []string) error {
 
 func (c *cli) listStacks(mgr *terramate.Manager, isChanged bool) ([]terramate.Entry, error) {
 	if isChanged {
+		log.Trace().
+			Str("action", "listStacks()").
+			Str("stack", c.wd()).
+			Msg("`Changed` flag was set. List changed stacks.")
 		return mgr.ListChanged()
 	}
-
 	return mgr.List()
 }
 
 func (c *cli) printStacks() error {
+	logger := log.With().
+		Str("action", "printStacks()").
+		Logger()
+
+	logger.Trace().
+		Str("stack", c.wd()).
+		Msg("Create a new stack manager.")
 	mgr := terramate.NewManager(c.root(), c.prj.baseRef)
+
+	logger.Trace().
+		Str("stack", c.wd()).
+		Msg("Get stack list.")
```

Comment:
We could add the working dir on the logger build here + we should not log c.wd() as a "stack", this is incorrect.